packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Changed
+  * Removed aria-checked to make screen readers announce Selected in `terra-form-select`.
+  * Updated focus to shift back on select upon key press or mouse click event while using Safari browser.
+
```

Comment:
Can this `Changed` section be combined with the `Changed` section on [lines 12-13](https://github.com/cerner/terra-core/pull/3811/files#diff-0d87855fd9582c564e6ca6a9654fcfa4099b7e717a0704a990d80316f05bdf59R12-R13)?